### PR TITLE
docs: Update README for running notebooks

### PR DESCRIPTION
**Pull request by manish0kuniyal**
Updated the python version from 3.7 to 3.10.
Provided description about running the notebook on VS Code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,23 @@ The following jupyter extensions are recommended but not required
 
 We have created a public [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace in Terra that contains the demo notebooks along with instructions for running them with minimal setup. To get started, see either the [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace or the [`Terra.ipynb`](notebooks/Terra.ipynb) notebook in this repository.
 
+## Running the Notebooks with VS Code
+[VS Code](https://code.visualstudio.com/) is a code editor developed by Microsoft. It is lightweight, highly customizable, and supports a wide range of programming languages, with a robust extension system.
+
+**1.Open VS Code**: Launch Visual Studio Code.   
+
+**2.Install Jupyter Extension**: Use Extensions view (Ctrl+Shift+X) to install the Jupyter extension.
```
Comment (korikuzma) [inline README.md:127]:
```suggestion
**2.Install Jupyter Extension**: Use Extensions view (Ctrl+Shift+X or ⌘+Shift+X) to install the [Jupyter extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
```

File: README.md [hunk 2]
```diff
@@ -119,6 +119,23 @@ The following jupyter extensions are recommended but not required
...
+
+**3.Open Project in VS Code**: Navigate to your project folder and open it in VS Code.   
+
+**4.Select Python Interpreter**: In a notebook, click "Select Python Interpreter" and choose your Python 3.10 virtual environment.
```
Comment (korikuzma) [inline README.md:131]:
```suggestion
**4. Select Jupyter Kernel**: In a notebook, click `Select Kernel` at the top right.  Select the option where the path is `venv/3.10/bin/python3`. See [here](https://code.visualstudio.com/docs/datascience/jupyter-kernel-management) for more information on managing Jupyter Kernels in VS Code.
```

File: README.md [hunk 3]
```diff
@@ -119,6 +119,23 @@ The following jupyter extensions are recommended but not required
 
 We have created a public [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace in Terra that contains the demo notebooks along with instructions for running them with minimal setup. To get started, see either the [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace or the [`Terra.ipynb`](notebooks/Terra.ipynb) notebook in this repository.
 
+## Running the Notebooks with VS Code
+[VS Code](https://code.visualstudio.com/) is a code editor developed by Microsoft. It is lightweight, highly customizable, and supports a wide range of programming languages, with a robust extension system.
+
+**1.Open VS Code**: Launch Visual Studio Code.
```
Comment (korikuzma) [inline README.md:125]:
```suggestion
**1. Open VS Code**: Launch Visual Studio Code.   
```

File: README.md (hunk 1 above)
Comment (korikuzma) [inline README.md:127]:
```suggestion
**2. Install Jupyter Extension**: Use Extensions view (Ctrl+Shift+X) to install the Jupyter extension.
```

File: README.md [hunk 4]
```diff
@@ -119,6 +119,23 @@ The following jupyter extensions are recommended but not required
...
 We have created a public [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace in Terra that contains the demo notebooks along with instructions for running them with minimal setup. To get started, see either the [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace or the [`Terra.ipynb`](notebooks/Terra.ipynb) notebook in this repository.
 
+## Running the Notebooks with VS Code
+[VS Code](https://code.visualstudio.com/) is a code editor developed by Microsoft. It is lightweight, highly customizable, and supports a wide range of programming languages, with a robust extension system.
+
+**1.Open VS Code**: Launch Visual Studio Code.   
+
+**2.Install Jupyter Extension**: Use Extensions view (Ctrl+Shift+X) to install the Jupyter extension.
+
+**3.Open Project in VS Code**: Navigate to your project folder and open it in VS Code.
```
Comment (korikuzma) [inline README.md:129]:
```suggestion
**3. Open Project in VS Code**: Navigate to your project folder and open it in VS Code.   
```

File: README.md [hunk 5]
```diff
@@ -119,6 +119,23 @@ The following jupyter extensions are recommended but not required
...
+**3.Open Project in VS Code**: Navigate to your project folder and open it in VS Code.   
+
+**4.Select Python Interpreter**: In a notebook, click "Select Python Interpreter" and choose your Python 3.10 virtual environment.  
+
+**5.Run Notebook**: After selecting the interpreter, run the notebook using "Run All Cells" or individual cells.
+
+**6.Ensure Kernel is Selected**: Check the top-right corner for the Jupyter kernel. If not set to the correct Python version, click and choose the desired kernel.    
+
+**7.Python Interpreter Not Selected**: If the Python interpreter is not selected, open a notebook file and click "Select Python Interpreter" again. Choose the Python 3.10 virtual environment you've set up for this project.  
+
```
Comment (korikuzma) [inline README.md:138]:
```suggestion
**5. Run Notebook**: After selecting the kernel you can now run the notebook.
```

**Review by korikuzma — CHANGES_REQUESTED** [review]

**Comment by manish0kuniyal** [comment]
The required changes have been made.

**Review by korikuzma — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -119,6 +119,18 @@ The following jupyter extensions are recommended but not required
 
 We have created a public [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace in Terra that contains the demo notebooks along with instructions for running them with minimal setup. To get started, see either the [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace or the [`Terra.ipynb`](notebooks/Terra.ipynb) notebook in this repository.
 
+## Running the Notebooks with VS Code
+[VS Code](https://code.visualstudio.com/) is a code editor developed by Microsoft. It is lightweight, highly customizable, and supports a wide range of programming languages, with a robust extension system.You can download VS Code [here](https://code.visualstudio.com/Download).
```
Comment (korikuzma) [inline README.md:123]:
```suggestion
[VS Code](https://code.visualstudio.com/) is a code editor developed by Microsoft. It is lightweight, highly customizable, and supports a wide range of programming languages, with a robust extension system. You can download VS Code [here](https://code.visualstudio.com/Download).
```

File: README.md [hunk 7]
```diff
@@ -119,6 +119,18 @@ The following jupyter extensions are recommended but not required
...
+
+**1. Open VS Code**: Launch Visual Studio Code.   
+
+**2. Install Jupyter Extension**: Use Extensions view (Ctrl+Shift+X or ⌘+Shift+X) to install the [Jupyter extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
+
+**3. Open Project in VS Code**: Navigate to your project folder and open it in VS Code.   
+
+**4. Select Jupyter Kernel**: In a notebook, click `Select Kernel` at the top right.  Select the option where the path is `venv/3.10/bin/python3`. See [here](https://code.visualstudio.com/docs/datascience/jupyter-kernel-management) for more information on managing Jupyter Kernels in VS Code.
```
Comment (korikuzma) [inline README.md:131]:
```suggestion
**4. Select Jupyter Kernel**: In a notebook, click `Select Kernel` at the top right. Select the option where the path is `venv/3.10/bin/python3`. See [here](https://code.visualstudio.com/docs/datascience/jupyter-kernel-management) for more information on managing Jupyter Kernels in VS Code. 
```

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -119,6 +119,18 @@ The following jupyter extensions are recommended but not required
 
 We have created a public [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace in Terra that contains the demo notebooks along with instructions for running them with minimal setup. To get started, see either the [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace or the [`Terra.ipynb`](notebooks/Terra.ipynb) notebook in this repository.
 
+## Running the Notebooks with VS Code
+[VS Code](https://code.visualstudio.com/) is a code editor developed by Microsoft. It is lightweight, highly customizable, and supports a wide range of programming languages, with a robust extension system.You can download VS Code [here](https://code.visualstudio.com/Download).
+
+**1. Open VS Code**: Launch Visual Studio Code.   
+
```
Comment (korikuzma) [inline README.md:126]: Remove spaces between number bullets

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (manish0kuniyal) [reply README.md:126]: Done

File: README.md [hunk 9]
```diff
@@ -119,6 +119,18 @@ The following jupyter extensions are recommended but not required
...
+**1.Open VS Code**: Launch Visual Studio Code.   
+
+**2.Install Jupyter Extension**: Use Extensions view (Ctrl+Shift+X or ⌘+Shift+X) to install the [Jupyter extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
+
+**3.Open Project in VS Code**: Navigate to your project folder and open it in VS Code.   
+
+**4.Select Jupyter Kernel**: In a notebook, click `Select Kernel` at the top right. Select the option where the path is `venv/3.10/bin/python3`. See [here](https://code.visualstudio.com/docs/datascience/jupyter-kernel-management) for more information on managing Jupyter Kernels in VS Code.  
+
+**5.Run Notebook**: After selecting the kernel you can now run the notebook.
```
[the remark this answers is not in the document]
Comment (manish0kuniyal) [reply README.md]: Hello again,

I noticed the documentation mentions feedback in pull requests, but is it necessary in this particular case? If so, how should I proceed with providing feedback on this specific pull request? 
Additionally, the necessary modifications in the README have been updated .

[the remark this answers is not in the document]
Comment (korikuzma) [reply README.md]: @manish0kuniyal feedback isn't necessary. Please accept the suggested changes to remove the new lines between the number bullets.

**Review by manish0kuniyal — COMMENTED** [review]
updated

File: README.md [hunk 10]
```diff
@@ -119,6 +119,17 @@ The following jupyter extensions are recommended but not required
 
 We have created a public [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace in Terra that contains the demo notebooks along with instructions for running them with minimal setup. To get started, see either the [`VRS-demo-notebooks`](https://app.terra.bio/#workspaces/terra-outreach/VRS-demo-notebooks) workspace or the [`Terra.ipynb`](notebooks/Terra.ipynb) notebook in this repository.
 
+## Running the Notebooks with VS Code
+[VS Code](https://code.visualstudio.com/) is a code editor developed by Microsoft. It is lightweight, highly customizable, and supports a wide range of programming languages, with a robust extension system. You can download VS Code [here](https://code.visualstudio.com/Download).
+
+**1. Open VS Code**: Launch Visual Studio Code.   
+
```
Comment (korikuzma) [inline README.md:126]:
```suggestion
```

File: README.md [hunk 11]
```diff
@@ -119,6 +119,17 @@ The following jupyter extensions are recommended but not required
...
+**2. Install the Jupyter Extension**: Use Extensions view (Ctrl+Shift+X or ⌘+Shift+X) to install the [Jupyter extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
+
```
Comment (korikuzma) [inline README.md:128]:
```suggestion
```

File: README.md [hunk 12]
```diff
@@ -119,6 +119,17 @@ The following jupyter extensions are recommended but not required
...
+**3. Open the Project in VS Code**: Navigate to your project folder and open it in VS Code.   
+**4. Select the Jupyter Kernel**: In a notebook, click `Select Kernel` at the top right. Select the option where the path is `venv/3.10/bin/python3`. See [here](https://code.visualstudio.com/docs/datascience/jupyter-kernel-management) for more information on managing Jupyter Kernels in VS Code.  
+
```
Comment (korikuzma) [inline README.md:131]:
```suggestion
```

**Review by korikuzma — CHANGES_REQUESTED** [review]
@manish0kuniyal missed these ones

**Review by korikuzma — APPROVED** [review]

**Comment by korikuzma** [comment]
@manish0kuniyal thanks for contributing!

**Comment by manish0kuniyal** [comment]
Thank you, ma'am, for being patient with me and explaining all the things step by step. I hope to contribute more to the Global Alliance for Genomics and Health.

**Pull request merged**